src/basic/log.c

```diff
@@ -434,6 +434,8 @@ static int write_to_console(
                 const char *func,
                 const char *buffer) {
 
+        static int dumb = -1;
```

Comment:
Not a big matter, but maybe better to cache in `getenv_terminal_is_dumb()`?

Reply:
Not sure, environment can change, I'd prefer to keep it in `write_to_console()`